### DISCUSSION TITLE
Add .cjs to known JavaScript file extensions

### DISCUSSION THREAD

**Discussion by tanerijun** [zed-industries/zed#6541] [Feature Requests]
Zed seems to recognize `.mjs` but not `.cjs`. Is there a way to fix this through config?

**Comment by hovsater** [comment]
Not right now. You can set the language manually for the current buffer, but it won’t persist upon close unfortunately.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
